### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review and give suggestions if any.

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    font-size: 1.5rem;
+    border: 0px;
+    padding: 1rem;
+    outline: 2px solid var(--GREY);
+    background-color: transparent;
+    transition: outline 0.3s ease-in;
+}
+
+.input-rounded {
+    border-radius: 999px;
```
Comment (zoef-yt) [inline components/input/input.css:17]: 50% would be enough i guess

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+.input-color-success {
+    outline: 2px solid var(--COLOR-SUCCESS);
+}
+.input-color-success + .input-message {
+    color: var(--COLOR-SUCCESS-DARK);
+}
+
+.input-color-warning {
+    outline: 2px solid var(--COLOR-WARNING-DARK);
+}
```
Comment (zoef-yt) [inline components/input/input.css:55]: add spacing

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    outline: 2px solid var(--COLOR-ERROR);
+}
+
+.input-color-error + .input-message {
+    color: var(--COLOR-ERROR-DARK);
+}
+
+.input-color-success {
+    outline: 2px solid var(--COLOR-SUCCESS);
+}
```
Comment (zoef-yt) [inline components/input/input.css:47]: add spacing

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by zoef-yt — APPROVED** [review]

**Pull request merged**